client/go.mod

```diff
@@ -31,11 +31,11 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/mod v0.6.0 // indirect
-	golang.org/x/net v0.1.1-0.20221027164007-c63010009c80 // indirect
+	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10 // indirect
```

Comment:
This one is causing the trivy scan failure. Is this golang.org/x/net v0.3.1 referenced in other sidecar's go.mod?
Can you check who is using this lib?
One way to workaround this problem is to manually add a replace statement:
replace (
  golang.org/x/net => v0.4.0
)

Reply:
Let me try to update this